prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -95,14 +97,16 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
             }))
           ),
         enrolleeNavigation: this.adjudicationResource.getAdjacentEnrolleeId(enrolleeId),
+        reviewStatus: this.adjudicationResource.getEnrolleeReviewStatus(enrolleeId)
```

Comment:
This is where i meant to make that comment, the reviewStatus is only being consumed by the "app-review-status-content" component so we can move this call to inside there.